phpseclib/Crypt/Rijndael.php

```diff
@@ -190,29 +190,6 @@ class Rijndael extends Base
     var $kl;
 
     /**
-     * Default Constructor.
```

Comment:
Hmmm, well, it's not broken at this point in the history: https://github.com/GrahamForks/phpseclib/blob/3f3e0cac98a479da04174c7805e4bcc390a27f0c/phpseclib/Crypt/Rijndael.php.